### PR TITLE
fixing tiny deprecated warnings

### PR DESCRIPTION
**Pull request by cesine**
there were two tiny deprecated warnings, with no apparent changes required in the code
https://github.com/atom/electron/blob/3875b30fcf2065cebc4e914ee5274084875e425b/atom/browser/api/lib/ipc.js#L5

seems to be deprecated around November 10 2015 https://github.com/atom/electron/commit/844cea8f216539ad2233180ab28b934d060f12c7

```bash
$ npm start

> playback@1.6.0 start ~/playback
> electron app.js

(electron) ipc module is deprecated. Use require("electron").ipcMain instead.
(electron) loadUrl is deprecated. Use loadURL instead.
```

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 var app = require('app')
 var BrowserWindow = require('browser-window')
 var path = require('path')
-var ipc = require('ipc')
+var ipc = require("electron").ipcMain
```
Comment (mafintosh) [inline app.js]: minor nitpick. could you use `'` instead of `"`?

File: index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ var drop = require('drag-and-drop-files')
 var mdns = require('multicast-dns')()
 var concat = require('concat-stream')
 var vtt = require('srt-to-vtt')
-var ipc = require('ipc')
+var ipc = require("electron").ipcMain
```
Comment (mafintosh) [inline index.js]: same as above

**Comment by mafintosh** [comment]
thanks :) i commented a minor style change

**Comment by cesine** [comment]
@mafintosh sorry about those ", fixed now

**Comment by mafintosh** [comment]
@cesine awesome. thank you!

**Comment by cesine** [comment]
thank you! this is an awesome example for what i'm working on :)

**Comment by cesine** [comment]
@tabrath thanks for catching this, i dug in and you're right: it needs to be `ipcRenderer` in the `index.js` and `ipcMain` in the `app.js`

**Comment by tabrath** [comment]
@cesine happy to help out.

**Pull request merged**